src/index.ts

```diff
@@ -15,3 +15,140 @@ export type {
   ApplicationFunction,
   ApplicationFunctionOptions,
 } from "./types";
+
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      /**
+       * The App ID assigned to your GitHub App.
+       * @example '1234'
+       */
+      APP_ID?: string;
+
+      /**
+       * By default, logs are formatted for readability in development. You can
+       * set this to `json` in order to disable the formatting.
+       */
+      LOG_FORMAT?: "json" | "pretty";
+
+      /**
+       * The verbosity of logs to show when running your app, which can be
+       * `fatal`, `error`, `warn`, `info`, `debug`, `trace` or `silent`.
+       * @default 'info'
+       */
+      LOG_LEVEL?:
+        | "trace"
+        | "debug"
+        | "info"
+        | "warn"
+        | "error"
+        | "fatal"
+        | "silent";
+
+      /**
+       * By default, when using the `json` format, the level printed in the log
+       * records is an int (`10`, `20`, ..). This option tells the logger to
+       * print level as a string: `{"level": "info"}`. Default `false`
+       */
+      LOG_LEVEL_IN_STRING?: "true" | "false";
+
```

Comment:
It looks like my review didn't get published, but one minor issue I have here is that these types are the expected input, but may not always be what is actually passed. Relying on the types now could cause issues when the input doesn't match the types.